addons/missionmodules/stringtable.xml

```diff
@@ -137,5 +137,13 @@
             <Czech>Smyčka okkolního zvuku (synchronizováno v MP)</Czech>
             <Portuguese>Loop de sons ambientes (sincronizados através do MP)</Portuguese>
         </Key>
+        <Key ID="STR_ACE_MissionModules_PlayerBreath_DisplayName">
+            <English>[ACE] Player Breathing</English>
```

Comment:
What's the purpose of `[ACE]` here, it's already under `ACE Mission Modules` category.